package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tezos-buidl",
   "private": true,
-  "description": "Interactive Code School for onboarding newcomers to the Tezos Ecosystem using smartpy",
+  "description": "Interactive Code School for onboarding new to the Tezos Ecosystem",
```

Comment:
Did we miss something here? (new developers/users?)

Reply:
Fixed. Thanks for the catch!